### PR TITLE
IDCommand: disallow tool prop to be blank

### PR DESCRIPTION
**Pull request by sevein**
This is connected to https://github.com/archivematica/Issues/issues/304.

### CODE REVIEW

File: fpr/migrations/0027_idcommand_tool_disallow_blank.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    dependencies = [
+        ('fpr', '0026_fits_nailgun_compat'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='idcommand',
+            name='tool',
+            field=models.ForeignKey(verbose_name='the related tool', to_field=b'uuid', to='fpr.IDTool', null=True),
```
Comment (ross-spencer) [inline fpr/migrations/0027_idcommand_tool_disallow_blank.py:17]: A small thing, but this line I think is 116 characters. It should be brought into line with our recommended coding style before being merged.

**Review by ross-spencer — APPROVED** [review]
Just the one comment to look at before merging, but LGTM.

File: fpr/migrations/0027_idcommand_tool_disallow_blank.py (hunk 1 above)
Comment (sevein) [reply fpr/migrations/0027_idcommand_tool_disallow_blank.py:17]: That code is generated by Django, do you still want me to do it?
Originally, we decided to leave it as is. flake8 was set up to exclude migrations, see:

https://github.com/artefactual/archivematica-fpr-admin/blob/230a99a18fe7557ca1a310d5a1827b0104505952/setup.cfg#L1-L5

Comment (ross-spencer) [reply fpr/migrations/0027_idcommand_tool_disallow_blank.py:17]: Thanks for the background @sevein. I say go ahead and merge :+1:

**Pull request merged**